.github/workflows/CI.yml

```diff
@@ -355,7 +355,19 @@ jobs:
         if: env.storage_models_changes_exist == 'true'
         run: cargo hack check --each-feature --no-dev-deps -p storage_models
 
+      - name: Decrypt connector auth file
+        run: ./decrypt_connector_auth.sh
+        env:
+          CONNECTOR_AUTH_PASSPHRASE: ${{ secrets.CONNECTOR_AUTH_PASSPHRASE }}
 
+      - name: Set connector auth file path in env
+        run: echo "CONNECTOR_AUTH_FILE_PATH=$HOME/target/test/connector_auth.toml" >> $GITHUB_ENV
+
+      - name: Run connector tests
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --package router --test connectors -- stripe
```

Comment:
Could we please use the shell command directly and keep it consistent with the rest of the steps in this workflow?